src/client/src/assets/appservice.svg

```diff
@@ -1 +1 @@
58XV+wdAYAAy0WwJ6fBhcd7zr/xaE9f9R8SSjHZf+DAboHscQ3OFBthTT6/Zco3aLZ29a+eEwan0rjS47cu6GYUGAFk3xVb2Nu58d6afB/6egcO/vovABL3uYUnt24RIBkrvapWamKkIa13qzB4dDMCNAr0+aYge6a9e3CFa75SNtK9rpk4S0D0tKroqdXRNF62LKXxfxTVlCi+K4IzHd2Dba7ZSnz55fA7AD51zcVVAqJHN8PreVQOARpFcLije3C3a7Zcu3btKqhN7mov9tETwR2umUmpXLSumQSKuNKjCohgT3v3wLOuuXJ98cXwPgCfuebiKPbRg0GDa2QyUWhHXTNJxOt5VCkj8vO24wPrXHPl2LVrVwGCw665OIp/9Ko0IpmLaesDV3k0VQZyqLVzoNk1Vw5bKh1wzcRR7KMnIlXdc9SzfkkhUu1zzSUJk0dTJcCCIJBDrrly5PP5DwAk7u9W7KMHxTLXiEuk6HfNJAkXelQNAVa1dw0+7Jorh1pN3Gov/tED7nINuFhBv01RKSLXAJGDMfJaS+9QzjXnUiqNvemaiZskRG/Rq6++Wt0fToQP0nSCm6aA00zRRXOj6GnXlMuOHTsuIWEfL0pC9NDU1HSXa2YydjQ8xdNOom8SI7W5hUWQqFfaJyJ6IpmqrusdfezuYYV84poj8okovvu9d/vuc8252JJO+dn4mZCI6JkQq10zLkWrJ10zRL7JBMGDrhkX1RJXejWnuN814qKqv+fzqkTfIqj6udzt27d/iAQ9nZGM6AErqz3MGA7MO2M2OX8wRNNBgOWbuj5a6JpzS85HyZMSvcbZs+dXtdrrWb+kUFIc4skn0Tc1IKzqmvm45DzumZToQQJd45pxKdnoaBreTpLmz1nS9AvEVPdSDwAQuegaiYvkRA9S9bWHoxuXnS5YTdQ9RTfD5FEtKbT66AGXXQNxkZjoAVj+xhtvVP2gdMnipaQfaCTpD40SwEjV1/RUddg1ExeJ+vtjwrDdNeNyeOPSd0ejZK/2uLulWjKQqld6GuGCayYuEhW9WmxxAaCkyBcT/K3YRP2hUeypVa8+Gxm6BqZMsP/JLVu2u8ZmwuENSz98/PjgsYyRza7ZOBo/yEhutClerGrVhxD5/NYDAGryxpU3Dx6s6xcavV00WMVLSV7tBdziUo0IknPyWgveRu/whqUfjgr2J/W+vZAX9qhGSjZi9HzxRbHwYkGT+ZRGyOZRDSgwcjWbSczTFLXgdfQ6N957aaQUbU7iNjcQ4f16VANyqmf9koJrKk28jh4wfsPy1cg+m8SP7WQMs0fVsda+65pJG++jBwCHNjT/19UIryXt+l6GzaMqlQL7nmsmbRi9CZ8ZeX7EIlHv3OMWl6oicua3Lcu9ukcPYPS+0rN+SWE4KnSMWCTqDcsN3OLSFNlI97pm0ojRu07nxnsvFSK7edRqYk50M4arPaqcAmcPb1zq3fU8gNG7weENSz+8WrQPjGpyVnw80KBKqcVLrpm0YvRu4uim5oGrX5hVoxFOJuFwI8vmUQUUOObrKg9g9G7p6GN3D/+fYP1ohNfifjuLEUGWqz0qgwo+ia6GeddcmjF6k+hZv6Rw8KF7fnilZJ+Le/iywpeL0uQUGCkWo46jj92dmHff1QOjV4bfbGh+5fPIPliI8QGHEeFJLk3Kwj7zvw8v/8A1l3aMXpmOPNR84m+lwvKC6r64rvoyRvhMLt1AgZEIdtuRh5bV5NVPScfoVaBz472X9j94zzPDY3bZiMWxOB5y5Ljao+so8JmFbmbwvsboTcHRTc0DB9Yv+ZfhsdLqQmTPxil+gQjDRwAABfpUsfbIQ80nXLM+YfSqcGTT8j/ub1n6j1dL2j4W4VPX/HTJcZvrNQVG1OInw8asOrxh6Yeued/U73XxHnlrw9IjAI58//jAuoYgbBfYtlCk0fXr6mlWYHA1sijFZxFK00AhvVFknz+6qXnANesrRq+G/md8G3Gi9e2Pn5s1K1ofBFhtIGsEumwmPtDdYAQjVpGGD5zTZORTK/pmEdHrPr5AoFKMXh1M3Ad1eOIHNnV9tPA2yT4QGqzUwKwIoXepYmW9Q2hE0GiAK5HyM0IposBngJyD6mlVnDu88R5vn66YCkZvGnRuvPcSgHcmftygtXOgeUGIO2/2c9USAIFVO+oapESIAr3A1Vx1GL0YmLj+wmswRNOAp7dE5BVGj4i8wugRkVcYPSLyCqNHRF5h9IjIK4weEXmF0SMirzB6ROQVRo+IvMLoEZFXGD0i8gqjR0ReYfSIyCt8tZTHDhx86w+ArnXNVcpGWJPPbzntmpvML37xi3nZhobnXHMVs7i8bdvWV1xjlF6MHsVSEATzBbLHNVcxg08AMHoe4/aWiLzC6BGRVxg9IvIKo0dEXmH0iMgrjB4ReYXRIyKvMHpE5BVGj4i8wugRkVcYPSLyCqNHRF5h9IjIK4weEXmF0SMirzB6ROQVRo+IvMLoEZFXGD0i8gqjR0ReYfSIyCuMHhF5hdEjIq8wekTkFUaPiLzC6BGRVxg9IvIKo0dEXmH0iMgrjB4ReYXRIyKvMHpE5BVGj4i8wugRkVcYPSLyCqNHRF5h9IjIK4weEXmF0SMirzB6ROQVRo+IvMLoEZFXGD0i8gqjR0ReYfSIyCuMHhF5hdEjIq8wekTkFUaPiLzC6BGRVxg9IvIKo0dEXmH0iMgrjB4ReYXRIyKvMHpE5BVGj4i8wugRkVcYPSLyCqNHRF5h9IjIK4weEXmF0SMirzB6ROSV0DUwZYr8gYO/zrvGiKZZw/79v17tGqKZY0JZDHVNTV39okcUT4tMgFOuIZpBdQwewO0tEXmG0SMirzB6ROQVRo+IvMLoEZFXGD0i8gqjR0ReYfSIyCuMHhF5hdEjIq8wekTkFUaPiLzC6BGRVxg9IvIKo0dEXmH0iMgrjB4ReYXRIyKvMHpE5BVGj4i88v+WO+bBhqZfmwAAAABJRU5ErkJggg=="/></svg>
```

Comment:
Accessibility items 
- change title to App service 
- add alt="App service logo" and role="img" in svg tag

Reply:
thank u accessibility champ

Reply:
actually it seems like you're doing it here #969 
https://github.com/microsoft/WebTemplateStudio/blob/e95b05c02fc85de61ce36a7665a5b2c6a9638f0d/src/client/src/assets/appservice.svg